index.html

```diff
@@ -632,7 +643,22 @@ <h4>The <code>displaystyle</code> and <code>scriptlevel</code> attributes</h4>
             and <code>false</code> and 2, respectively.
           </div>
         </section>
-      </section>
+        <section id="the-intent-and-arg-attributes">
```

Comment:
Is this too specific and this should just be called "Attributes Reserved as Valid" or something?
Could it just say something like 

> While MathML Core Level 1 defines no behavior for these attributes, and no functionality is currently implied by this specification, validators must treat them as valid.   Future versions of this specification may or may not define them. Authors should be aware that they are currently in development and subject to change.  I also have a comment about alttext specifically that I will put in the PR

Reply:
used this text exactly